lib/transaction/index.js

```diff
@@ -149,6 +149,11 @@ function Transaction(agent) {
   this.isDistributedTrace = null
   this.acceptedDistributedTrace = null
 
+  // LLM fields.
+  this.llm = {
+    conversation_id_tracker: new Map()
```

Comment:
nit: this seems very wordy.  maybe just `this.llm.responses` as technically it's not tracked by conversation id but from request id. I'm also not sure how this will get serialized, it's prob ok but i'll dig a little more

Reply:
Seems good to me. I just couldn't think of a better name.